src/World.cpp

```diff
@@ -520,6 +520,40 @@ void cWorld::Start(void)
 		m_LinkedOverworldName = IniFile.GetValueSet("LinkedWorlds", "OverworldName", GetLinkedOverworldName());
 	}
 
+	// If we are linked to one or more worlds that do not exist, ask the server to stop.
+	AString BadWorld = "";
+	cRoot * Root = cRoot::Get();
+	if (GetDimension() == dimOverworld)
+	{
+		if ((m_LinkedNetherWorldName != "") && (Root->GetWorld(m_LinkedNetherWorldName) == nullptr))
+		{
+			BadWorld = m_LinkedNetherWorldName;
+		}
+		if ((m_LinkedEndWorldName != "") && (Root->GetWorld(m_LinkedEndWorldName) == nullptr))
+		{
+			if (BadWorld != "")
+			{
+				BadWorld += ", ";
+			}
+			BadWorld += m_LinkedEndWorldName;
+		}
+	}
+	else
+	{
+		if ((m_LinkedOverworldName != "") && (Root->GetWorld(m_LinkedOverworldName) == nullptr))
+		{
+			BadWorld = m_LinkedOverworldName;
+		}
+	}
+	if (BadWorld != "")
+	{
+		const char * WorldName = m_WorldName.c_str();
+		LOGD("\n######\nERROR: %s is linked to one or more invalid worlds: %s\nPlease edit %s/world.ini and fix this.\n######\n",
```

Comment:
I don't think it's possible, and also not exactly necessary, the error message is colored so it stands out in the console. Having the log sequential makes things easier to understand, too, when reading the log, you know exactly what went wrong where.